### ISSUE TITLE
Unable to reproduce paper accuracy using the provided pre-trained weights

### ISSUE THREAD

**Issue by Bellakira7** [vigil1917/Analytical-Gaze-Generalization-framework#3]
<img width="611" height="185" alt="Image" src="https://github.com/user-attachments/assets/8fe1ccd6-fe7b-4982-a256-fa499dc48819" />
Hi authors,

First of all, thank you for the excellent paper and for open-sourcing the code and pre-trained weights! It is a very inspiring work.

I am currently trying to evaluate the model using the pre-trained weights provided in your repository to reproduce the cross-domain results. However, the Mean Angular Error (MAE) I get is noticeably higher than the reported accuracy.

**Comment by vigil1917** [comment]
Hi! Thank you for your interests in our work!
I have reproduced the performance of the open-source weights multiple times locally, and the implementation should be correct.
For a small-scale dataset like MPII, an error of 16.58 degrees is clearly abnormal.
My suggestion is to check whether the gaze definition is consistent with the model outputs using a distribution plot (with ground‑truth yaw/pitch on the x‑axis and predicted values on the y‑axis). It should give you some clue.

**Comment by Bellakira7** [comment]
> 嗨！感谢您对我们工作的关注！我已经多次在本地复现了开源权重的性能，实现应该是正确的。对于像MPII这样的小规模数据集，16.58度的误差显然是异常的。我的建议是检查凝视定义是否与模型输出一致，使用分布图（x轴为地面偏航/俯仰，y轴为预测值）。这应该能给你一些线索。

<img width="3570" height="1466" alt="Image" src="https://github.com/user-attachments/assets/75da70ab-8be2-4f6d-8642-05c09e9e367f" />
This is the result I obtained. Could you please tell me what caused this?

**Comment by Bellakira7** [comment]
At the same time, I'm not sure if I've made a mistake in my data processing. I noticed that all the datasets in your code have the same header (face gaze2d head2d origin gaze3d head3d face gaze head origin cam_index frame_index normmat). Could you provide the code for this processing?
